src/providers/DevicesProvider/AudioOutputProvider.tsx

```diff
@@ -17,7 +17,7 @@ import { useMeetingManager } from '../MeetingProvider';
 
 const AudioOutputContext = createContext<AudioOutputContextType | null>(null);
 
-const AudioOutputProvider: React.FC = ({ children }) => {
+const AudioOutputProvider: React.FC<React.PropsWithChildren<unknown>> = ({ children }) => {
```

Comment:
Wonder if we can drop the `unknown`? In some newer version community reported that only `React.PropsWithChildren` are enough, I may be wrong.

Reply:
Yeah I think we can drop it. This change is also done by script. Probably because its type is `any` and `any` is not recommended anymore so it's `unknown` now.